### PR TITLE
Add tutorial for basic command line interactions

### PR DESCRIPTION
**Pull request by ptare**
Signed-off-by: Prasad Tare <prasad.tare@gmail.com>

### CODE REVIEW

File: documentation/tutorials/customizing-tokens-via-cli.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## Token creation
+
+First launch truffle console:
+
+```
+$(npm bin)/truffle console --network=<networok>
+```
+
+Pass `--network=test` for local runs. We'll type the rest of the commands in this document into the Truffle console.
```
Comment (mrice32) [inline documentation/tutorials/customizing-tokens-via-cli.md]: Should we tell them what network to use for rinkeby?

File: documentation/tutorials/customizing-tokens-via-cli.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+*true*
+```
+
+We're all set the create the token now. Note that there are large number of customization options for
+`TokenizedDerivative`: this tutorial only scratches the surface.
+
+```
+> const priceFeed = await ManualPriceFeed.deployed()
+> const noLeverageCalculator = await LeveragedReturnCalculator.deployed()
+> const params = { priceFeedAddress: priceFeed.address, defaultPenalty: web3.utils.toWei("0.5", "ether"), supportedMove: web3.utils.toWei("0.1", "ether"), product: web3.utils.utf8ToHex("BTC/USD"), fixedYearlyFee: web3.utils.toWei("0.01", "ether"), disputeDeposit: web3.utils.toWei("0.5", "ether"), returnCalculator: noLeverageCalculator.address, startingTokenPrice: web3.utils.toWei("1", "ether"), expiry: 0, marginCurrency: "0x0000000000000000000000000000000000000000", withdrawLimit: web3.utils.toWei("0.33", "ether"), returnType: "1", startingUnderlyingPrice: "0", name: "Name", symbol: "SYM" }
```
Comment (mrice32) [inline documentation/tutorials/customizing-tokens-via-cli.md]: Are there any line breaks that we can put here that the console will accept to make this more readable? For instance, in bash, you can escape the newlines by ending lines with `\`.

Also, do you think we should add parameters of interest on separate lines (like `product` and `marginCurrency`)? So we can call them out specifically?

File: documentation/tutorials/customizing-tokens-via-cli.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Customizing tokens via CLI
+
+In this tutorial, we'll dig into more details and go beyond what the dApp allows.
+We'll customize the parameters of synthetic tokens via the command line by creating a BTC token
```
Comment (mrice32) [inline documentation/tutorials/customizing-tokens-via-cli.md]: Do you think we should refer to it as a BTC/USD token or do you think that would make the quanto/compo thing more confusing?

File: documentation/tutorials/customizing-tokens-via-cli.md (hunk 1 above)
Comment (ptare) [reply documentation/tutorials/customizing-tokens-via-cli.md]: Sure, makes sense.

File: documentation/tutorials/customizing-tokens-via-cli.md (hunk 2 above)
Comment (ptare) [reply documentation/tutorials/customizing-tokens-via-cli.md]: No line breaks as far as I was able to figure out.

File: documentation/tutorials/customizing-tokens-via-cli.md (hunk 3 above)
Comment (ptare) [reply documentation/tutorials/customizing-tokens-via-cli.md]: Either way, what do you prefer?

File: documentation/tutorials/customizing-tokens-via-cli.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+margined in ETH and interacting with it via the command line.
+
+## Financial engineering note
+
+The financial engineering of this token is _complicated_, because the creator is exposed to both Bitcoin and ETH price moves.
```
Comment (rcai1) [inline documentation/tutorials/customizing-tokens-via-cli.md]: Maybe we should add 4 bullet points here: 
- If the price of BTC/USD is 10,000, the token facility will require that at least (10,000 * Collateralization Ratio) ETH are deposited in the token facility. If the Collateralization Ratio is 1.25, the token facility will require that at least 10,000 * 1.25 = 12,500 ETH are in the token facility. 
- If the price of BTC/USD goes from 10,000 to 12,000, the token facility will require that at least (12,000 * Collateralization Ratio) ETH are deposited in the token facility. If the Collateralization Ratio is 1.25, the token facility will require that at least 12,000 * 1.25 = 15,000 ETH are in the token facility. 
- If the price of BTC/USD goes from 10,000 to 8,000, the token facility will require that at least (8,000 * Collateralization Ratio) ETH are deposited in the token facility. If the Collateralization Ratio is 1.25, the token facility will require that at least 8,000 * 1.25 = 10,000 ETH are in the token facility. 
- The token facility will require these minimum collateralization amounts, no matter what the price of ETH/USD is.

File: documentation/tutorials/customizing-tokens-via-cli.md (hunk 3 above)
Comment (ptare) [reply documentation/tutorials/customizing-tokens-via-cli.md]: Done.

File: documentation/tutorials/customizing-tokens-via-cli.md (hunk 4 above)
Comment (ptare) [reply documentation/tutorials/customizing-tokens-via-cli.md]: Done.

**Pull request merged**